docs/en/observability/cloud-monitoring/azure/monitor-azure-openai-apm.asciidoc

```diff
@@ -0,0 +1,204 @@
+[discrete]
+[[azure-openai-apm]]
+=== Step 6: Monitor Microsoft Azure OpenAI APM with OpenTelemetry
+
+The Azure OpenAI API provides useful data to help monitor and understand your code.
+Using OpenTelemetry, you can ingest this data into Elastic {observability}.
+From there, you can view and analyze your data to monitor the cost and performance of your applications.
+
+For this tutorial, we'll be using an https://github.com/mdbirnstiehl/AzureOpenAIAPMmonitoringOtel[example Python application] and the Python OpenTelemetry libraries to instrument the application and send data to {observability}.
```

Comment:
Can you create anew directory in https://github.com/elastic/observability-examples and move the sample application here.